library/src/blas_ex/rocblas_gemm_ex.hpp

```diff
@@ -683,7 +691,14 @@ rocblas_status gemm_ex_batched_template(rocblas_handle    handle,
         handle, trans_a,  trans_b, m,    n,   k,        alpha, a,   lda,      stride_a,   b,
         ldb,    stride_b, beta,    c_in, ldi, stride_i, d,     ldd, stride_d, batch_count};
 
-    return handle->host->runContractionProblem(problem);
+    if(startEvent && stopEvent)
+    {
+        hipStream_t stream;
+        rocblas_get_stream(handle, &stream);
+        return handle->host->runContractionProblem(problem, &stream, startEvent, stopEvent);
```

Comment:
`runContractionProblem`  has access to the handle, so we do not need to pass `stream` as a separate argument, unless you want to support arbitrary streams independent of `handle`.

Reply:
In `runContractionProblem`, simply use:
```
hipStream_t stream;
rocblas_get_stream(problem.handle, &stream);
```
and then you have `stream.

Reply:
good catch, I'll make the changes